chart/env/prod.yaml

```diff
@@ -139,7 +139,7 @@ envVars:
       },
       {
         "name": "Qwen/QwQ-32B-Preview",
-        "preprompt": "You are a helpful and harmless assistant. You are Qwen developed by Alibaba. You should think step-by-step.",
+        "preprompt": "",
```

Comment:
@nsarrazin or can I remove this field entirely?

Reply:
You can just remove it entirely!